src/viam/app/app_client.py

```diff
@@ -1370,7 +1370,7 @@ async def get_fragment(self, fragment_id: str) -> Fragment:
             # Get a fragment and print its name and when it was created.
             the_fragment = await cloud.get_fragment(
                 fragment_id="12a12ab1-1234-5678-abcd-abcd01234567")
-            print("Name: ", the_fragment.name, "\nCreated on: ", the_fragment.created_on)
+            print("Name: ", the_fragment.name, "\\nCreated on: ", the_fragment.created_on)
```

Comment:
Not sure if this works, is this how to escape newlines in code examples? Alternatively, likely safe to remove the newline. As is it [breaks formatting](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient.get_fragment).